pioinstaller/__main__.py

```diff
@@ -40,10 +38,9 @@ def cli(ctx, verbose, shutdown_piohome):
     log.debug("sys.version: %s", sys.version)
```

Comment:
Please also log.info(`Installer version: %s`) and change `log.debug` for this line to `log.info`. let show this info by default